elements/rh-video/rh-video.css

```diff
@@ -0,0 +1,180 @@
+:host {
+  display: flex;
+}
+
+:host([hidden]),
+*[hidden] {
+  display: none !important;
+}
+
+#video,
+#consent {
+  display: inline-flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+#video {
+  justify-content: stretch;
+  position: relative;
+}
+
+figure,
+::slotted([slot="thumbnail"]),
+::slotted(iframe) {
+  max-width: 100%;
+}
+
+::slotted([slot="thumbnail"]) {
+  display: block;
+}
+
+::slotted(iframe) {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  inset-block-start: 0;
+  inset-inline-start: 0;
+  border: 0;
+}
+
+figure {
+  --_video-focus-border-color: var(--rh-color-border-interactive-on-light, #0066cc);
+  --_video-play-btn-bkg-color: rgb(31 31 31 / var(--rh-opacity-50, 50%));
+  --_video-play-btn-interactive-bkg-color: rgb(21 21 21 / var(--rh-opacity-80, 80%));
+
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+figure.dark {
+  --_video-focus-border-color: var(--rh-color-border-interactive-on-dark, #92c5f9);
+  --_video-play-btn-bkg-color: rgb(255 255 255 / 0.25);
+  --_video-play-btn-interactive-bkg-color: rgb(255 255 255 / 0.55);
```

Comment:
@adamjohnson @bennypowers, should these colors still use our opacity token values?

Reply:
I thought about changing them to:

```css
rgb(255 255 255 / var(--rh-opacity-20, 20%))
rgb(255 255 255 / var(--rh-opacity-50, 50%))
```
But 25% and 55% were what was specified in [Figma](https://www.figma.com/design/X4BB7K26HMPBL8GfKbQvpw/Element---Video-(WIP)?node-id=0-1&t=qktZrt5Pzs6kR5R9-1). Do you want me to change them to their token values (listed above)?

Reply:
Yeah, that sounds good! I've created [this issue](https://github.com/RedHat-UX/red-hat-design-system/issues/1705) to update the video Figma component too.

Reply:
Resolved in 6a8a0e5.